_observability-plugin/trace/trace-analytics-jaeger.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: default
+title: Analyze Jaeger trace data 
+parent: Trace analytics
+nav_order: 55
+---
+
+# Analyze Jaeger trace data
+
+Introduced 2.5
+{: .label .label-purple }
+
+Trace analytics now supports Jaeger trace data. Jaeger can store its trace data in OpenSearch, but users that use OpenSearch as their backend for Jaeger data can now benefit from trace analytics capability.
+
+When you perform trace analytics, you can select from two data sources:
+
+- **Data Prepper** – Data ingested into OpenSearch through Data Prepper. This data source requires OpenTelemetry (OTEL) index type.
+- **Jaeger** – Trace data stored within OpenSearch as its backend. This data source requires the Jaeger index type.
+
+If you currently store your Jaeger trace data in OpenSearch, you can now use the trace analytics capability. When you ingest Jaeger data into OpenSearch, it gets stored in a different index than the OTA-generated index that gets created when you run data through the Data Prepper. You can indicate which data source on which you want to perform trace analytics with the data source selector in the Dashboards.
+
+Jaeger trace data that you can analyze includes span data, as well as service and operation endpoint data. Jaeger span data analysis requires some configuration.
+
+Each time you ingest data for Jaeger, it creates a separate index for that day. The Dashboards will show the current index that has a mapping.
+
+To learn more about Jaeger data tracing, see the [Jaeger](https://www.jaegertracing.io/) open source documentation.
+
+## Requirements
+
+Jaeger data that is ingested for OpenSearch needs to have the `--es.tags-as-fields.all=true` flag set for errors. If data is not ingested in this format it will not work for trace analytics with OpenSearch.
+
+### About Data ingestion with Jaeger indexes
+
+Trace analytics for non-Jaeger data use OTEL indexes with the naming conventions `otel-v1-apm-span-*` or `otel-v1-apm-service-map*`.
+
+Jaeger indexes follow the naming conventions `jaeger-span-*` or `jaeger-service-*`.
+
+Jaeger and OTEL indexes have different field names. Therefore when you run trace analytics, you'll need to create different queries and components depending on which index type you are using.
+{:.note}
```

Comment:
What does this mean? We are creating the dashboard for them?

Reply:
will comment this out for now. this info was in the feature brief or one of the eng tickets.

Reply:
i hid it with comments if we need it for 2.6 when they provide the details about those.